cli/src/commands/get.rs

```diff
@@ -176,7 +176,16 @@ pub fn run(get_args: &GetArgs, client: Client, printer: &Printer) -> Result<()>
                 });
                 sources
             };
-            printer.print_resources(&sources)?;
+            let mut enriched_sources: Vec<EnrichedSource> = Vec::with_capacity(sources.len());
+            for source in sources.into_iter() {
+                let bucket = match source.bucket_id.as_ref() {
+                    // User may not have permission to fetch bucket, or bucket may not exist
+                    Some(bucket_id) => client.get_bucket(bucket_id.clone()).ok(),
```

Comment:
Here we're making a separate API call for every source, which is not ideal. Instead we can use `client.get_buckets()` to produce a `HashMap<BucketId, Bucket>` and use that to associate buckets with sources.